test/e2e/storage/persistent_volumes-local.go

```diff
@@ -265,6 +265,46 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 				})
 			})
 
+			Context("Set fsGroup for local volume", func() {
+
+				It("should enforce fsGroup for one pod", func() {
+					fsGroup := int64(1234)
+					pod, err := createLocalPod(config, testVol, &fsGroup)
+					framework.ExpectNoError(err)
+					By("Checking fsGroup is enforced")
+					_, err = framework.LookForStringInPodExec(config.ns, pod.Name, []string{"ls", "-ld", volumeDir}, strconv.Itoa(int(fsGroup)), time.Second*10)
```

Comment:
You may get false positives if the fsGroup number shows up elsewhere in the output.  It's probably better to split and get the word you need.

Reply:
@msau42 Switched to use `stat -c '%g'` to get group id only. Same as `ls`, it is in coreutils. We can depend on it.